x/emissions/keeper/inference_synthesis/network_regrets.go

```diff
@@ -392,14 +445,13 @@ func CalcTopicInitialRegret(regrets []alloraMath.Dec, epsilon alloraMath.Dec, pN
 		return alloraMath.ZeroDec(), err
 	}
 
-	minimumRegret := alloraMath.ZeroDec()
-	for i, regret := range regrets {
-		if i == 0 || regret.Lt(minimumRegret) {
-			minimumRegret = regret
-		}
+	// Calculate percentile
+	percentile, err := alloraMath.GetQuantileOfDecs(regrets, alloraMath.MustNewDecFromString("0.25"))
```

Comment:
should this 25% be a module parameter or topic parameter?

Reply:
@kpeluso I think we need a new emissions module parameter for this, I think this should not be hardcoded to 25%

Reply:
still need an answer @kpeluso , I am not happy about merging this without at least acknowledgement

Reply:
it's added as module param

Reply:
@kpeluso @kpeluso @kpeluso @kpeluso @kpeluso @kpeluso @kpeluso @kpeluso @kpeluso @kpeluso @kpeluso @kpeluso @kpeluso @kpeluso @kpeluso @kpeluso @kpeluso @kpeluso @kpeluso @kpeluso @kpeluso @kpeluso @kpeluso @kpeluso

Reply:
@spooktheducks I convinced RedBird to do it already, it should be in this PR now

Reply:
indeed afaict